src/components/Post.js

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import axiosInstance from "../api/axios";
+import img1 from "../images/img1.jpg";
+import { useNavigate } from "react-router-dom";
+
+
+const Post = () => {
+  const [blog, setBlog] = useState({ title: "", content: "", author: "" });
+  let navigate = useNavigate();
+
+  const getBlogDetails = (e) => {
```

Comment:
Any name with `handle` in it is a better name

Reply:
ok